docs/clients/iquart_flight_controller_interface.rst

```diff
@@ -0,0 +1,132 @@
+A minimal working example for the IQUartFlightControllerInterfaceClient using the telemetry endpoint is:
+
+.. include:: ../notes/c-full-code.rst
+
+.. code-block:: c++
+
+    #include "generic_interface.hpp"
+    #include "iquart_flight_controller_interface_client.hpp"
+    #include <iostream>
+
+    void main(){
+        // Make a communication interface object
+        GenericInterface com;
+
+        // Make a IQUART Flight Controller Interface object with obj_id 0
+        IQUartFlightControllerInterfaceClient iquartFlightControllerInterface(0);
+
+        // Use the IQUART Flight Controller Interface Client
+        iquartFlightControllerInterface.telemetry_.get(com)
+
+        // Insert code for interfacing with hardware here  
+
+        // Store telemetry data in an IFCITelemetryData object
+        IFCITelemetryData telemtry = iquart_flight_controller_interface.telemetry_.get_reply()
+
+        // Examples on how to access each property of IFCITelemetryData
+        cout << "telemetry coil_temp: " << telemetry.coil_temp << endl;
+        cout << "telemetry consumption: " << telemetry.consumption << endl;
+        cout << "telemetry current: " << telemetry.current << endl;
+        cout << "telemetry mcu temp: " << telemetry.mcu_temp << endl;
+        cout << "telemetry speed: " << telemetry.speed << endl;
+        cout << "telemetry uptime: " << telemetry.uptime << endl;
+        cout << "telemetry voltage: " << telemetry.voltage << endl;
+    }
+
+Matlab
+~~~~~~
+
+To use the IQUART Flight Controller Interface client in Matlab, all Vertiq communication code must be included in your
+path. This allows the creation of a IQUartFlightControllerInterfaceClient object. See the Message Table below for available messages.
+All message strings use the Short Names. All messages use the standard Get/Set/Save functions.
+
+A minimal working example for the IQUartFlightControllerInterfaceClient is:
+
+.. code-block:: Matlab
+
+    % Make a communication interface object
+    com = MessageInterface(’COM18’,115200);
+    % Make a IQUartFlightControllerInterfaceClient object with obj_id 0
+    iquartFlightControllerInterface = IQUartFlightControllerInterfaceClient(’com’,com);
+    % Use the IQUartFlightControllerInterfaceClient object
+    pulsingVoltageMode = iquartFlightControllerInterface.get(’telemetry’);
+
+Python
+~~~~~~
+
+To use the IQUART Flight Controller Interface Client in Python, import ``iqmotion`` and create a module that has the Arming Handler Client within its firmware. 
+See the table below for available messages. All message strings use the Short Names. 
+All messages use the standard Get/Set/Save functions.
+
+A minimal working example for the IQUART Flight Controller Interface Client is:
+
+.. code-block:: Python
+    :substitutions:
+    
+    import iqmotion as iq
+
+    com = iq.SerialCommunicator("/dev/ttyUSB0")
+    |variable_name| = iq.|module_name|(com, 0)
```

Comment:
|variable_name| and |module_name| are showing up as those literal strings.  They should be filled in.